USAGE.md

```diff
@@ -68,38 +68,51 @@ URL" input allowing you to open a layer json from a remote source.
-the current version, as described below.
-
-The layer upgrade interface displays the technique changes that occurred between the two versions. These are grouped by the
-following change types:
-- <b>Additions:</b> new techniques that have been added since the previous version
-- <b>Changes:</b> changes to techniques
-- <b>Minor Changes:</b> changes to techniques without version increments
-- <b>Deprecations:</b> techniques which have been deprecated
-- <b>Revocations:</b> techniques which have been revoked by a new technique
-- <b>Unchanged:</b> techniques with no changes between the two versions
-
-By default, only techniques which were previously annotated are shown in the interface. The filters at the top of the interface
-can be used to individually adjust which change types are shown.
+The layer upgrade interface allows users to upgrade an ATT&CK Navigator layer created on a previous version of ATT&CK to the 
+current version of the dataset. With this interface, users can step through each change in the dataset of techniques:
+- The user can view techniques which have been added to the dataset since the layer was created and annotate them as necessary.
+- The user can see whether the techniques they had previously annotated have changed and adjust the annotations accordingly.
+- The user can determine if any annotated techniques have been removed or replaced by new techniques and in the latter case copy 
+annotations to the replacing technique(s).
+- The user can also verify what techniques haven't changed since the layer was created.
+
+### Change Types
+
+The workflow steps through the changes to the dataset that occurred between the two versions, grouped by the following change types:
+- <b>Additions:</b> New techniques that have been added to the knowledge base.
+- <b>Changes:</b> Techniques with major updates such as changes to scope, new reporting, and so forth.
+- <b>Minor Changes:</b> Techniques with minor updates such as typo corrections.
+- <b>Revocations:</b> Techniques which have been replaced by other techniques.
+- <b>Deprecations:</b> Techniques which have been removed from the dataset.
+- <b>Unchanged:</b> Techniques which have not changed between the two versions.
 
 ### Mapping Annotations
 
-Each technique section contains cells for each tactic it is found under in the previous version (with any annotations that
-were previously placed on that object) and in the current version. There are two approaches to copying annotations from the
-previous layer:
+For annotated techniques which have changed since the layer was created, you may want to update the set of annotations to account 
+for changes to scope or content. Each technique in the interface lists its previous and current state, with links to 
+<a href="https://attack.mitre.org/">ATT&CK&reg;</a> for both versions of the technique to enable easy review. In steps with techniques you 
```

Comment:
ATT&CK hyperlink here should be "the ATT&CK Website" instead of just "ATT&CK." ATT&CK is the knowledge base, the ATT&CK Website is the tool used to interface with it.

Also the registered trademark is only required for the first occurrence of ATT&CK on a given page. In this case the header at the top of the help page covers the trademark.